compiler/bcq-tools/generate_bcq_output_arrays.py

```diff
@@ -255,6 +255,196 @@ def print_bcq_output_arrays(flags):
         raise SystemExit(err_msg)
 
 
+def get_bcq_version(input_path):
```

Comment:
Why is this added?

Reply:
@seanshpark print_* method will be deprecated. Because it returns its output as a file, which is redundant if we use python script. So, get_* methods return its output as python things.